src/TrackGenerator.cpp

```diff
@@ -843,7 +894,7 @@ void TrackGenerator::initializeTracks() {
     for (int j = 0; j < _num_y[i]; j++) {
 
       /* If Track points to the upper right */
-      if (i < _num_azim / 2)
+      if (i < _num_azim / 4)
```

Comment:
In some places you use spaces when you divide `_num_azim` by 2 or 4, and in other places you do not. Just pick a standard and be consistent throughout (all files, not just this one).